aruco_ros/src/aruco_detector.cpp

```diff
@@ -316,45 +327,66 @@ class ArucoSimple
     arucoMsg.error_code = error_condition;
     arucoMsg.error_message = error_message;
     arucoMsg.pose = poseMsg;
+
+    // Include the corner points of the marker, so we an reconstruct its position in the UI
+    for (int i = 0; i < 4; ++i) {
+      aruco_msgs::Corner corner;
+      corner.x = marker[i].x;
+      corner.y = marker[i].y;
+      arucoMsg.corners.push_back(corner);
+    }
     pose_pub.publish(arucoMsg);
   }
 
   void image_callback(const sensor_msgs::ImageConstPtr& msg)
   {
+    unsigned int error_code;
     ros::Time curr_stamp(ros::Time::now());
     if(cam_info_received)
     {
       cv_bridge::CvImagePtr cv_ptr;
-
       try
       {
         cv_ptr = cv_bridge::toCvCopy(msg, sensor_msgs::image_encodings::RGB8);
         inImage = cv_ptr->image;
 
+        if (is_camera_rotated) { cv::rotate(inImage, inImage, cv::ROTATE_90_CLOCKWISE); }
```

Comment:
multiple lines please

Reply:
also, in in place rotations work?

Reply:
My thoughts at first too. Looking at the OCV function, looks like they're copying data behind the scenes